modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/item/selector/ObjectEntryItemSelectorView.java

```diff
@@ -275,9 +275,9 @@ public SearchContainer<ObjectEntry> getSearchContainer()
 			try {
 				Page<com.liferay.object.rest.dto.v1_0.ObjectEntry>
 					objectEntriesPage = _objectEntryManager.getObjectEntries(
-						_themeDisplay.getCompanyId(), _objectDefinition, null,
-						null, _getDTOConverterContext(), _getFilterString(),
-						null, null, null);
+						_themeDisplay.getCompanyId(), _objectDefinition,
+						StringPool.BLANK, null, _getDTOConverterContext(),
```

Comment:
I believe this shouldn't be `BLANK`, because it's supposed to pass the `groupKey` of the Site the user is accessing.

Reply:
Hi @rodrigopaulino 
Thanks for reviewing it!
I haven't done that test, thanks for catching it!
Would you find a good fix to use themeDisplay.getScopeGroup().getGroupKey() as groupKey instead of blank?
If you agree, I'll add a commit with that change.
Thank you very much!

Reply:
I have tried that solution in my local and it works rightly

Reply:
It looks appropriate to me, @lfbesada. ☺️

Reply:
Done, thanks a lot!